### ISSUE TITLE
Feature Request (Add Favourites Field)

### ISSUE THREAD

**Issue by hentai-chan** [moe-ka/nhentai.py#1]
# Feature Request
Doujins on nhentai have a favorite button that indicates its popularity. It would be great if this could be added to the doujin dictionary as a new field. Here's a code snippet that finds this number:

```python
import requests
import nhentai

doujin = nhentai.get_doujin(328334)
url = f"https://nhentai.net/g/{doujin.id}/"

response = requests.get(url)
favorites = response.content.decode("utf-8").find("nobold")
# returns '6583' at the time of writing (2020/09/10)
print(favorites) 
``` 

I also noticed that the doujin dictionary doesn't provide the URL to the doujin's main page (`url`), but only its individual (media) pages? Ideally, the implementation for both features would use the function 

```python
def _get(endpoint, params={}):
	return _SESSION.get("https://nhentai.net/api/" + endpoint, params=params).json()
```

but I wasn't sure which endpoints I should use for making this request [1]. It's certainly not a good idea to make a second request with `requests.get(url)` just to add these fields.

## References
- [1] https://nhentai.net/info/

**Comment by hysrx** [comment]
Heyo! So I did some reverse engineering and did the following based on the _get function

```python
>>> import requests
>>> sesh = requests.Session()
>>> resp = sesh.get('https://nhentai.net/api/gallery/177013', params={})
>>> resp.json()
```

And the return of `resp.json()` has a key called "num_favorites", so regarding favourite implementation, shouldnt be too difficult and would not require an additional `requests.get(url)`.

**Comment by hentai-chan** [comment]
Nice! This is a much better solution! I didn't know about the `api/gallery` path, and reading a JSON response is much safer parsing HTML. I suggest you make an pull request :)

**Comment by hysrx** [comment]
Sure! Also regarding retrieval of the URL, unless major overhauls of the site occur its pretty safe to use `nhentai.net/g/` then the given ID.

**Comment by moe-ka** [comment]
thank you for the pr and issue! 
merged #2
